dev-python/sympy/sympy-1.11.1-r1.ebuild

```diff
@@ -30,7 +30,7 @@ RDEPEND="
 		png? ( app-text/dvipng )
 		pdf? ( app-text/ghostscript-gpl )
 	)
-	mathml? ( dev-libs/libxml2:2[${PYTHON_USEDEP}] )
+	mathml? ( dev-python/lxml[${PYTHON_USEDEP}] )
```

Comment:
RDEPEND changes require a new revision, but this one is obviously correct, so we can do `git mv` rather than dropping to `~arch`: change your not-yet-in-tree -r2 to -r3 and then
`git my sympy-1.11.1-r1.ebuild sympy-1.11.1-r2.ebuild`.

Reply:
Thanks (I had done this before being told this already yesterday), done.